packages/tree-widget/src/components/TreeUiItemsProvider.tsx

```diff
@@ -0,0 +1,127 @@
+import { I18N } from "@bentley/imodeljs-i18n";
+import {
+  AbstractWidgetProps,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+} from "@bentley/ui-abstract";
+import {
+  UiFramework,
+  SpatialContainmentTree,
+  ClassGroupingOption
+} from "@bentley/ui-framework";
+import React from "react";
+import { TreeWidgetComponent } from "./TreeWidgetComponent";
+import { CategoriesTreeComponent, ModelsTreeComponent, TreeWidgetControlOptions } from "../tree-widget-react";
```

Comment:
Could you import these 3 from the actual module and not the barrel file?

Reply:
Pretty sure we'll hit circular import issues; or atleast we do in other pkgs when we import things from the same pkg from the barrel